docs/getting-started.md

```diff
@@ -55,6 +55,15 @@ Your change is now published to the corresponding PR. You and your colleagues ar
 
 ## Installation
 
+### Homebrew users
+
+To install maiao with homebrew, you need to tap the repo before installing it:
+
+```bash
+brew tap adevinta/maiao https://github.com/adevinta/maiao.git
+brew install adevinta/maiao/maiao
```

Comment:
Two questions:

1. Why don't we submit this in the [homebrew/core](https://docs.brew.sh/Adding-Software-to-Homebrew#submitting-the-formula) tap?
2. Is there a way to avoid a double `/maiao/maiao` ?

Reply:
Not that I know, but this would be a great improvement.
You can test it with my personal [fork](https://github.com/tjamet/maiao/) 

```
brew tap tjamet/maiao https://github.com/adevinta/maiao.git
brew install tjamet/maiao/maiao
```

I didn't find a way to specify the branch name for the tap, making it harder to test (hence the fork on main branch)
Let me know if you want to test it :D or whether we consider this as a future improvement :D

Reply:
For me it's OK as it is. If we can submit it to homebrew/core, that's great too.

Reply:
```suggestion
brew tap adevinta/maiao https://github.com/adevinta/maiao.git
brew install maiao
```

Just tried and it worked, as soon as you have tapped the repo, it is included in the lookup